dotnet/src/Connectors/Connectors.GoogleVertexAI/Connectors.GoogleVertexAI.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- THIS PROPERTY GROUP MUST COME FIRST -->
+    <AssemblyName>Microsoft.SemanticKernel.Connectors.GoogleVertexAI</AssemblyName>
+    <RootNamespace>$(AssemblyName)</RootNamespace>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <VersionSuffix>alpha</VersionSuffix>
+    <NoWarn>SKEXP0001,SKEXP0070</NoWarn>
+  </PropertyGroup>
+
+  <!-- IMPORT NUGET PACKAGE SHARED PROPERTIES -->
+  <Import Project="$(RepoRoot)/dotnet/nuget/nuget-package.props"/>
+  <Import Project="$(RepoRoot)/dotnet/src/InternalUtilities/src/InternalUtilities.props"/>
+
+  <PropertyGroup>
+    <!-- NuGet Package Settings -->
+    <Title>Semantic Kernel - Google AI/Vertex AI connectors</Title>
+    <Description>Semantic Kernel connectors for Google Gemini. Contains clients for text generation and text embedding generation.</Description>
```

Comment:
@matthewbolanos Suggestion for the description and package name.

This package supports both GoogleAI and VertexAI platforms with current naming is not clear it also supports `GoogleAI`

### Package name suggestions
- keep `Microsoft.SemanticKernel.Connectors.GoogleVertexAI`
- change to `Microsoft.SemanticKernel.Connectors.Google`

### Description change

This change leaves the space open for adding more services without having to revisit the description every time a new modality is added.

```suggestion
    <Description>Semantic Kernel connectors for Google generation platforms (GoogleAI/VertexAI). Contains generation and embedding services.</Description>
```

Reply:
Are you sure? I also think current name is too long. But just "google" will be clear? So all new features related to google have to be in this package.

Reply:
Namespace also?

Reply:
I spoke with @matthewbolanos, the idea is the Connector to have the name of the provider and any service within will be provided by the same package for the given provider. 

If we end up needing to have multiple projects for one provider we can always create a Meta Package that combines multiple platforms of a same provider.